CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.19.0 (13/05/2024)
+
+**Common**
+* Added `Promote` to the `olp::cache::KeyValueCache` API to explicitly promote the key in the LRU cache.
+* Fixed the CMake configuration for MSVC build.
+* Updated code snippets and the documentation to reflect recent changes in API.
+
+**olp-cpp-sdk-authentication**
+* Changed `olp::authentication::AuthenticationClientImpl::SignInClient` to retry basing on the `olp::client::RetrySettings` instead of returning error immediately in case of transient errors.
```

Comment:
```suggestion
* Changed `olp::authentication::AuthenticationClientImpl::SignInClient` to retry based on the configuration in `olp::client::RetrySettings` instead of returning an error immediately in case of transient errors.
```